### ISSUE TITLE
DR | Profile Contact info missing telephone extension

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#72457]
## Value Statement
**_As a_** Veteran viewing the contact info page within the form
**_I want to_** be able to see the telephone extension I entered
**_So that_** I know it has been included without needing to edit the telephone

## Background Context

Within the platform `ContactInfo` component, a telephone number is rendered without including any extension value.

## Acceptance Criteria

- [x] Include any telephone extension on the contact info page
- [ ] Unit test added to verify the extension is visible

## Tasks
- [ ] Update the platform `ContactInfo` component to render telephone extension
- [ ] Add unit test

## Definition of Ready 
- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Reviewed and approved by product and/or design

**Comment by Mottie** [comment]
Fix merged into component library. Now we need to wait for the component library update before we can test.

**Comment by Mottie** [comment]
Work merged in and ready for testing in staging @saderagsdale @eileen-coforma

**Comment by eileen-coforma** [comment]
@Mottie @saderagsdale 
Extension now shows on the contact info. It's hidden if there's no response. Also gets updated in Profile if updated is made in form. This works properly.

**Comment by eileen-coforma** [comment]
@Mottie 
Not sure why, but the mobile phone number slim success alert is now persisting for the form after I restarted the form.

https://github.com/department-of-veterans-affairs/va.gov-team/assets/111457818/6cd66699-5a89-4c28-a974-dda809fe0db9

**Comment by Mottie** [comment]
I'm working on a PR to fix this now 😰

**Comment by Mottie** [comment]
Hmm, actually... if you click on the continue this application later link, it completely bypasses all code to clear the updated alert flag. I'm gonna have to dive into platform code to fix this :cracks-knuckles:
